README.md

```diff
@@ -61,12 +61,14 @@ Integrating this library into a site takes several steps. For the sake of comple
 ### Create the Drupal Site and Initialize DDEV
 
 ```console
-composer create-project drupal/recommended-project pwtest
-composer require drush/drush
+mkdir pwtest
 cd pwtest
-ddev config --project-type drupal10
-ddev get deviantintegral/ddev-playwright
+ddev config --project-type=drupal --php-version=8.3 --docroot=web
```

Comment:
I wonder if we should omit the PHP version, so we don't have to change this as time goes on.

Reply:
If people have a recent ddev version should be ok omitting it.